crates/radix-engine-toolkit/Cargo.toml

```diff
@@ -8,14 +8,13 @@ build = "build.rs"
 # radixdlt-scrypto dependencies.
 sbor = { workspace = true }
 scrypto = { workspace = true }
-transaction = { workspace = true }
+radix-common = { workspace = true }
```

Comment:
Let's bump the version to `2.1.0-dev1` since this is not compatiable with either mainnet or stokenet

Reply:
I'm getting `ld: malformed 64-bit a.b.c.d.e version number: 2.1.0-dev1` error on my MacOS.

Reply:
Ok, figured this out, it was due to custom `build.rs` for MacOS which adds library version through `ld` linker additional param. It is not mandatory so I can remove that part.